code/modules/client/verbs/looc.dm

```diff
@@ -0,0 +1,89 @@
+#define LOOC_RANGE 7
```

Comment:
**(OPTIONAL)**
If you want to give LOOC a colour in chat rather than just being black:
<hr>

**Before:**
![Before](https://user-images.githubusercontent.com/57483089/167643247-ce9cc6e2-d968-4dc3-89bf-02527856da92.png)
**After:**
![After](https://user-images.githubusercontent.com/57483089/167643288-7785144a-9509-4e23-9832-00391279d3f6.png)
<hr>

`tgui\packages\tgui-panel\styles\goon\chat-light.scss`
```diff
.ooc {
  color: #002eb8;
  font-weight: bold;
}

+.looc {
+  color: #6699CC;
+}
+
.adminobserverooc {
```
`tgui\packages\tgui-panel\styles\goon\chat-dark.scss`
```diff
.ooc {
  color: #cca300;
  font-weight: bold;
}

+.looc {
+  color: #6699CC;
+}
+
.adminobserverooc {
```